stdlib/socketserver.pyi

```diff
@@ -6,17 +6,18 @@ from typing import Any, BinaryIO, Callable, ClassVar, Union
 
 _RequestType = Union[_socket, tuple[bytes, _socket]]
 _AddressType = Union[tuple[str, int], str]
+_RequestHandlerClassCallable = Callable[[_RequestType, _AddressType, BaseServer], BaseRequestHandler]
```

Comment:
There's a new primer error, because using unions as argument types to `Callable`  has the same problem as returning a union. Users should be prepared to handle only the one union member that will actually be returned (or passed as argument), not all of them.

Apparently the type of `client_address` depends on what `get_request()` returns. At runtime, `get_request()` isn't even defined in `BaseServer`, but only in subclasses. For example, TCPServer returns `socket.socket.accept()`, which is currently `Any`-typed. Maybe `BaseServer` should be generic?